code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -188,24 +188,30 @@
 		msg += "[t_He] [p_do()]n't seem all there.\n"
 
 	if(temp)
-		if(temp < 30)
+		if (temp < 25)

```

Comment:
Stahp.

Reply:
Spaces between ifs and parentheses are a crime against humanity.